Godeps/Godeps.json

```diff
@@ -406,6 +406,11 @@
 			"Rev": "a1b8ba5163b7f041b22761461eabd02b70d1f824"
 		},
 		{
+			"ImportPath": "github.com/gogo/protobuf/proto",
+			"Comment": "v0.2-17-gc18eea6",
```

Comment:
With `cni` we are in same situation for about a month or so (for 0.3.0 release). Maybe we should wait in similar way for `proto`?
In `cni` now is an additional rush (and i have for this additional resources), maybe `protobuf` needs same interest?